hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -99,6 +95,12 @@ private static BitSet validateAndCopy(BitSet acls) {
     return copyBitSet(acls);
   }
 
+  private static BitSet validateAndCopy(EnumSet<ACLType> enumSet) {
+    BitSet aclBitSet = new BitSet();
+    enumSet.forEach(aclType -> aclBitSet.set(aclType.ordinal()));
+    return validateAndCopy(aclBitSet);
+  }
```

Comment:
- `BitSet` needs to be copied only when provided by external code (one that creates `OzoneAcl`).
- We can reuse `bitSetOf(ACLType...)` to create the `BitSet`.  One of the methods can call the other, e.g. `Arrays.asList(acls)` (and declare this method to accept any `Collection<ACLType>`) or `enumSet.toArray(new ACLType[0])`.